pages/index.js

```diff
@@ -48,10 +45,11 @@ export default function Home(data) {
 }
 
 export const getStaticProps = async () => {
-  // const response = await fetch(  "https://corona-stats.mobi/api/json.2.0.php?key=UTSiraH8NBz3JblhOcVI"
-  const data = rawData;
+  const response = await fetch("https://corona-stats.mobi/api/json.2.0.php?key=UTSiraH8NBz3JblhOcVI")
```

Comment:
You can do `const response = await fetch("/api/data")` if you add an api route as explained on salck